firstvoices/backend/resources/dictionary.py

```diff
@@ -92,17 +112,17 @@ class DictionaryEntryCategoryResource(BaseResource):
     class Meta:
         model = DictionaryEntryCategory
 
-    def before_import_row(self, row, **kwargs):
-        # Skip rows with categories that don't exist
+    def before_import_row(self, row, row_number=None, **kwargs):
+        # skip rows with non-existent categories
         logger = logging.getLogger(__name__)
-
         try:
             Category.objects.get(id=row["category"])
         except Category.DoesNotExist:
             logger.warning(
-                f"Skipping row with category id {row['category']} because it does not exist"
+                f"Skipping row {row_number} because entry {row['category']} does not exist."
             )
-            raise self.skip_row("Category does not exist")
+            row["category"] = ""
+            # TODO: prevent the row from being imported
```

Comment:
I could not quite figure out how to skip importing the row, if someone wants to pick this ticket up this is the only work left to do. Thanks!